models/changeRequest.js

```diff
@@ -1,4 +1,3 @@
-/* eslint-disable camelcase */
```

Comment:
In general I would say that all the removal of eslint lines are not positive changes. They are there because if not, the linter complains. This for example
![image](https://user-images.githubusercontent.com/2612426/65671868-5764c600-e048-11e9-9361-df5d313b2cb2.png)

But there were other changes like this one. For the eslint line to disappear, in this case it would also need to be changed to use camel case, but I don't recommend it because it changes the format of the response... So my suggestion would be to leave the eslint lines as they were

Reply:
I just noticed that my eslint was disabled. I'll reinclude the exception comments

Reply:
Implemented changes